libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -531,6 +532,11 @@ class SRV_Channels {
     // initialize before any call to push
     static void init();
 
+    // return true if a channel is set to type GPIO
+    static bool is_GPIO(uint8_t channel) {
```

Comment:
maybe change this to "is_disabled".. although I don't feel strongly about it.

Reply:
for how it is used I think is_GPIO() is clearer